code/about.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html id="top">
+
+<head>
+    <meta charset="ut-8" />
+    <meta name="viewport" content="width=device-width">
+    <title>About Us</title>
+    <link rel="stylesheet" href="./style.css" />
+</head>
+
+<header class="simple-header">
+    <nav class="nav">
+        <input type="checkbox" id="checkbox">
+        <label for="checkbox"><a class="menu-button"></a>
+        </label>
+        <div class="menu-content">
+            <a href="index.html" target="blank">home</a>
```

Comment:
Great idea to implement subpages! Nonetheless, proper value for `target` attribute is `_blank`, not `blank` ;)